src/main/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDao.kt

```diff
@@ -0,0 +1,67 @@
+package com.hibob.academy.employeeFeedback.dao
+
+import jakarta.ws.rs.BadRequestException
+import org.jooq.DSLContext
+import org.jooq.RecordMapper
+import org.jooq.Record
+import org.springframework.stereotype.Repository
+
+@Repository
+class FeedbackDao(private val sql: DSLContext) {
+
+    private val feedbackTable = FeedbackTable.instance
+
+    private val feedbackMapper = RecordMapper<Record, FeedbackInfo> { record ->
+        FeedbackInfo(
+            record[feedbackTable.id],
+            record[feedbackTable.employeeId],
+            record[feedbackTable.companyId],
+            record[feedbackTable.dateOfFeedback],
+            AnonymityType.convertStringToAnonymityType(record[feedbackTable.anonymity]),
+            record[feedbackTable.reviewed],
+            record[feedbackTable.feedback]
+        )
+    }
+
+    fun submitFeedback(feedbackToSubmit: FeedbackForSubmission): Int {
```

Comment:
don't change it, this is for general knowledge. we usually use Long for ids. This is because we are assuming that the system will grow and at some point int won't be enough for the ids

Reply:
Ok, i thought because  in the script we got it's serial and not big serial then it's int and not long.
Noted for next time.

Reply:
if you had serial in the script then you are correct.
This comment was for general knowledge